### ISSUE TITLE
BUG: After adding image to clusterfile,no interactive prompt

### ISSUE THREAD

**Issue by LZiHaN** [labring/sealos#3007]
### Sealos Version

main

### How to reproduce the bug?

basic Clusterfile

```
apiVersion: apps.sealos.io/v1beta1
kind: Cluster
metadata:
  name: default
spec:
  image:
  - hub.sealos.cn/labring/kubernetes:v1.25.6
  - hub.sealos.cn/labring/helm:v3.11.0
  - hub.sealos.cn/labring/flannel:v0.21.4
```

apply

`sealos apply -f Clusterfile `

add new app `hub.sealos.cn/labring/nginx:v1.23.3` to Clusterfile

```
apiVersion: apps.sealos.io/v1beta1
kind: Cluster
metadata:
  name: default
spec:
  image:
  - hub.sealos.cn/labring/kubernetes:v1.25.6
  - hub.sealos.cn/labring/helm:v3.11.0
  - hub.sealos.cn/labring/flannel:v0.21.4
  - hub.sealos.cn/labring/nginx:v1.23.3
```

apply again, program is blocked

### What is the expected behavior?

nginx installation complete.

### What do you see instead?

program is blocked.
`2023-04-27T15:16:12 info Executing ConfirmOverrideApps Pipeline in InstallProcessor` Stopped here.

```
root@sealos-test:/test/sealos-test# sealos apply --debug -f Clusterfile
2023-04-27T15:16:12 debug loadClusterFile loadRenderValues: map[]
2023-04-27T15:16:12 debug rendered Clusterfile: apiVersion: apps.sealos.io/v1beta1
kind: Cluster
metadata:
  name: default
spec:
  image:
  - hub.sealos.cn/labring/kubernetes:v1.25.6
  - hub.sealos.cn/labring/helm:v3.11.0
  - hub.sealos.cn/labring/flannel:v0.21.4
  - hub.sealos.cn/labring/nginx:v1.23.3

2023-04-27T15:16:12 debug loadClusterFile loadRenderValues: map[]
2023-04-27T15:16:12 debug rendered Clusterfile: apiVersion: apps.sealos.io/v1beta1
kind: Cluster
metadata:
  creationTimestamp: "2023-04-27T07:14:15Z"
  name: default
spec:
  hosts:
  - ips:
    - 192.168.64.8
    roles:
    - master
  image:
  - hub.sealos.cn/labring/kubernetes:v1.25.6
  - hub.sealos.cn/labring/helm:v3.11.0
  - hub.sealos.cn/labring/flannel:v0.21.4
  ssh: {}
status:
  commandCondition:
  - images:
    - hub.sealos.cn/labring/kubernetes:v1.25.6
    - hub.sealos.cn/labring/helm:v3.11.0
    - hub.sealos.cn/labring/flannel:v0.21.4
    lastHeartbeatTime: "2023-04-27T07:14:15Z"
    message: Applied to cluster successfully
    reason: Apply Command
    status: "True"
    type: ApplyCommandSuccess
  conditions:
  - lastHeartbeatTime: "2023-04-27T07:14:15Z"
    message: Applied to cluster successfully
    reason: Ready
    status: "True"
    type: ApplyClusterSuccess
  mounts:
  - env:
      criData: /var/lib/containerd
      defaultVIP: 10.103.97.2
      disableApparmor: "false"
      registryConfig: /etc/registry
      registryData: /var/lib/registry
      registryDomain: sealos.hub
      registryPassword: passw0rd
      registryPort: "5000"
      registryUsername: admin
      sandboxImage: pause:3.8
    imageName: hub.sealos.cn/labring/kubernetes:v1.25.6
    labels:
      auth: auth.sh
      check: check.sh $registryData
      clean: clean.sh && bash clean-cri.sh $criData
      clean-registry: clean-registry.sh $registryData $registryConfig
      image: ghcr.io/labring/lvscare:v4.1.6
      init: init-cri.sh && bash init.sh
      init-registry: init-registry.sh $registryData $registryConfig
      io.buildah.version: 1.28.1
      org.opencontainers.image.description: kubernetes-v1.25.6 container image
      org.opencontainers.image.licenses: MIT
      org.opencontainers.image.source: https://github.com/labring-actions/cache
      sealos.io.type: rootfs
      sealos.io.version: v1beta1
      version: v1.25.6
      vip: $defaultVIP
    mountPoint: /var/lib/containers/storage/overlay/f446971c79475e35a495f509e4de5a60f8ff8ef9519978908930b550c50f7f56/merged
    name: default-rn5f5jsy
    type: rootfs
  - cmd:
    - cp -a opt/helm /usr/bin/
    imageName: hub.sealos.cn/labring/helm:v3.11.0
    labels:
      io.buildah.version: 1.28.1
    mountPoint: /var/lib/containers/storage/overlay/00da0d4e3488b71fe76f5fcdee72c38b574237769459ad5e4a3efd60a8d51a5b/merged
    name: default-h9cd45t3
    type: application
  - cmd:
    - bash entrypoint.sh
    imageName: hub.sealos.cn/labring/flannel:v0.21.4
    labels:
      io.buildah.version: 1.28.1
    mountPoint: /var/lib/containers/storage/overlay/3ef9b905bf3366a2730150e865047ece49e05e1b4d224c25f0e237214ffd0991/merged
    name: default-aezf5xvq
    type: application
  phase: ClusterSuccess

2023-04-27T15:16:12 debug run new images: [hub.sealos.cn/labring/nginx:v1.23.3]
2023-04-27T15:16:12 info start to install app in this cluster
2023-04-27T15:16:12 info Executing SyncStatusAndCheck Pipeline in InstallProcessor
2023-04-27T15:16:12 debug parse reference a5fd294ad99fd9e83bd60a5efabd05d7a342ca91eda012ef7561257b047f71af with transport containers-storage
2023-04-27T15:16:12 debug parse reference 34018b130618f340f685ea8a30dec40a74b846a5a80cfd905666b837dcf3b50a with transport containers-storage
2023-04-27T15:16:12 debug parse reference 88e8e96d5c07d1490039da94109818db43a111b43db3e18ab95aa9d2978f6f01 with transport containers-storage
2023-04-27T15:16:12 debug sync cluster status is: apiVersion: apps.sealos.io/v1beta1
kind: Cluster
metadata:
  creationTimestamp: null
  name: default
spec:
  hosts:
  - ips:
    - 192.168.64.8
    roles:
    - master
  image:
  - hub.sealos.cn/labring/kubernetes:v1.25.6
  - hub.sealos.cn/labring/helm:v3.11.0
  - hub.sealos.cn/labring/flannel:v0.21.4
  - hub.sealos.cn/labring/nginx:v1.23.3
  ssh: {}
status:
  mounts:
  - env:
      criData: /var/lib/containerd
      defaultVIP: 10.103.97.2
      disableApparmor: "false"
      registryConfig: /etc/registry
      registryData: /var/lib/registry
      registryDomain: sealos.hub
      registryPassword: passw0rd
      registryPort: "5000"
      registryUsername: admin
      sandboxImage: pause:3.8
    imageName: hub.sealos.cn/labring/kubernetes:v1.25.6
    labels:
      auth: auth.sh
      check: check.sh $registryData
      clean: clean.sh && bash clean-cri.sh $criData
      clean-registry: clean-registry.sh $registryData $registryConfig
      image: ghcr.io/labring/lvscare:v4.1.6
      init: init-cri.sh && bash init.sh
      init-registry: init-registry.sh $registryData $registryConfig
      io.buildah.version: 1.28.1
      org.opencontainers.image.description: kubernetes-v1.25.6 container image
      org.opencontainers.image.licenses: MIT
      org.opencontainers.image.source: https://github.com/labring-actions/cache
      sealos.io.type: rootfs
      sealos.io.version: v1beta1
      version: v1.25.6
      vip: $defaultVIP
    mountPoint: /var/lib/containers/storage/overlay/f446971c79475e35a495f509e4de5a60f8ff8ef9519978908930b550c50f7f56/merged
    name: default-rn5f5jsy
    type: rootfs
  - cmd:
    - cp -a opt/helm /usr/bin/
    imageName: hub.sealos.cn/labring/helm:v3.11.0
    labels:
      io.buildah.version: 1.28.1
    mountPoint: /var/lib/containers/storage/overlay/00da0d4e3488b71fe76f5fcdee72c38b574237769459ad5e4a3efd60a8d51a5b/merged
    name: default-h9cd45t3
    type: application
  - cmd:
    - bash entrypoint.sh
    imageName: hub.sealos.cn/labring/flannel:v0.21.4
    labels:
      io.buildah.version: 1.28.1
    mountPoint: /var/lib/containers/storage/overlay/3ef9b905bf3366a2730150e865047ece49e05e1b4d224c25f0e237214ffd0991/merged
    name: default-aezf5xvq
    type: application
  phase: ClusterInProcess

2023-04-27T15:16:12 info Executing ConfirmOverrideApps Pipeline in InstallProcessor
```

### Operating environment

```markdown
- Sealos version:
- Docker version:
- Kubernetes version:
- Operating system:
- Runtime environment:
- Cluster size:
- Additional information:
```

### Additional information

https://github.com/labring/sealos/blob/d528d6be713b9b9cf92169e5822d354d29fffb9d/pkg/apply/processor/install.go#L103

seems to be here.

**Comment by LZiHaN** [comment]
#2993 
caused by this pr.

```
  - cmd:
    - cp -a opt/helm /usr/bin/
    imageName: hub.sealos.cn/labring/helm:v3.11.0
    labels:
      io.buildah.version: 1.28.1
    mountPoint: /var/lib/containers/storage/overlay/00da0d4e3488b71fe76f5fcdee72c38b574237769459ad5e4a3efd60a8d51a5b/merged
    name: default-h9cd45t3
    type: application
  - cmd:
    - bash entrypoint.sh
    imageName: hub.sealos.cn/labring/flannel:v0.21.4
    labels:
      io.buildah.version: 1.28.1
    mountPoint: /var/lib/containers/storage/overlay/3ef9b905bf3366a2730150e865047ece49e05e1b4d224c25f0e237214ffd0991/merged
    name: default-aezf5xvq
    type: application
  phase: ClusterInProcess

2023-04-27T15:16:12 info Executing ConfirmOverrideApps Pipeline in InstallProcessor
>> invalid input, please enter 'y', 'yes', 'n', or 'no'
2023-04-27T16:00:03 debug write cluster file to local storage: /root/.sealos/default/Clusterfile
Error: ^C
```

**Comment by nowinkeyy** [comment]
Is there an interactive prompt for the first apply? I recreate the process, and both apply have interactive prompts: ```Warning: Using an even number of master nodes is a risky operation and can lead to reduced high availability and potential resource wastage. " +
		"It is strongly recommended to use an odd number of master nodes for optimal cluster stability. " +
		"Are you sure you want to proceed?```

**Comment by nowinkeyy** [comment]
Hi, @LZiHaN 
Can you use ```sealos rmi labring/nginx:v1.23.3``` to remove the image and try to apply again?

**Comment by nowinkeyy** [comment]
Hi, @LZiHaN 
Can you use ```sealos rmi labring/nginx:v1.23.3``` to remove image and apply again? I tried many times, but there was no blocking.

**Comment by LZiHaN** [comment]
> Hi, @LZiHaN Can you use `sealos rmi labring/nginx:v1.23.3` to remove image and apply again? I tried many times, but there was no blocking.

Maybe you need to re-execute the above steps after `sealos reset`.

**Comment by LZiHaN** [comment]
> 1. sealos reset & sealos rmi -f images
> 2. sealos apply -f Clusterfile
> 3. add hub.sealos.cn/labring/nginx:v1.23.3 to Clusterfile & sealos apply -f Clusterfile
> 
> first apply: ![5fc2687702d8284acf5a4b96ed576ba](https://user-images.githubusercontent.com/72536832/235091284-55fe5047-6bf4-49f0-bd96-bdc1ca2bad70.png)
> 
> second apply: ![image](https://user-images.githubusercontent.com/72536832/235091581-87949f2e-8352-4182-83c5-950fba516abb.png)
> 
> I'm sorry I can't reproduce the problem. But another problem was found, prompting too many times.

`no interactive prompt ` this happens when I use.

**Comment by LZiHaN** [comment]
> 1. sealos reset & sealos rmi -f images
> 2. sealos apply -f Clusterfile
> 3. add hub.sealos.cn/labring/nginx:v1.23.3 to Clusterfile & sealos apply -f Clusterfile
> 
> first apply: ![5fc2687702d8284acf5a4b96ed576ba](https://user-images.githubusercontent.com/72536832/235091284-55fe5047-6bf4-49f0-bd96-bdc1ca2bad70.png)
> 
> second apply: ![image](https://user-images.githubusercontent.com/72536832/235091581-87949f2e-8352-4182-83c5-950fba516abb.png)
> 
> I'm sorry I can't reproduce the problem. But another problem was found, prompting too many times.

maybe you can fix your known problems, let me troubleshoot and modify this bug.

**Comment by LZiHaN** [comment]
fixed #3020
